Makefile

```diff
@@ -1,19 +1,23 @@
-SHELL = /bin/bash
+SHELL             := /bin/bash
+.SHELLFLAGS       += -e -u -o pipefail
 
-export NAME       ?= mailserver-testing:ci
-export IMAGE_NAME := $(NAME)
+export IMAGE_NAME := mailserver-testing:ci
+export NAME       ?= $(IMAGE_NAME)
```

Comment:
What's the difference between NAME/IMAGE?

Would be either one of them not enough?

Reply:
On paper, yes; in the future, yes; currently, no :) I want to migrate from `NAME` to `IMAGE_NAME`, but as of now, not all tests use `IMAGE_NAME`. I will resolve this later. We could add a comment though, if requested.